openapi.yaml

```diff
@@ -226,6 +226,50 @@ paths:
               schema:
                 $ref: '#/components/schemas/Problem'
 
+  /help:
+    get:
+      summary: Retrieve help information for the given command and any number of subcommands.
+      operationId: help
+      tags:
+        - help
+      parameters:
+        - name: command
+          in: query
+          style: form
+          explode: true
+          description: The command (and any subcommands) for which help information is needed. Subcommands can be separated by '/' or space ' '.
+          required: true
+          schema:
+            type: string
+      responses:
+        "200":
+          description: Help information retrieved successfully.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
```

Comment:
@princerajpoot20  can you update this schema to the correct response we are sending? Basically the two scenarios, where we have the all the command list, and then one command response. The command response might need to be generic (e.g. make it an object) since it changes for each endpoint, and it would take some time for you to do it on all commands.

For example you can do:
```yaml
schema:
  oneOf:
    - $ref: '#/components/schemas/HelpListResponse'
    - $ref: '#/components/schemas/HelpCommandResponse'
```